tools/src/main/scala/scala/scalanative/build/Build.scala

```diff
@@ -54,7 +54,6 @@ object Build {
     config.logger.time("Total") {
```

Comment:
I am not super excited about this because we are changing the build for such a specific purpose unless there are other specific use cases in mind.

Refactoring out the "findAndCompileNativeSources" is good but perhaps instead of using `fold` for the special case you could have `match` / `case` with `None` being first as the normal case and then hiding the body of the `fold` in another method.

I think this would be more readable and show intent better. Also, I think that the `// compile generated ll` should go first in that block before the "findAndCompileNativeSources". It logically follows closer to the code just above.

WDYT?

Reply:
You're right, I've replaced it with a proper match.
 I think this change might be useful for us in other aspects, eg. we can try to add some elements of incremental compilation to our build, for now, it can be just native sources, not much, but it still might save a few seconds.